README.md

```diff
@@ -1,62 +1,73 @@
-
-| Name                   | Description                                                         |
-| ---------------------- | ------------------------------------------------------------------- |
-| KEYCLOAK_SERVER        | URL to the keycloak server to use                                   |
-| WEBSOCKET_URL          | URL to the websocket to listen to                                   |
-| WEBHOOK_TOKEN          | Token to use to authenticate with websocket                         |
-| OKDATA_CLIENT_SECRET   | Client Secret of the keycloak client to use                         |
-| OKDATA_CLIENT_ID       | Client ID of the keycloak client to use                             |
-| OKDATA_ENVIRONMENT     | Which environment to run on(dev/prod)                               |
-| KEYCLOAK_REALM         | Name of the keycloak realm to use                                   |
-| AWS_XRAY_SDK_ENABLED   | Whether to use AWS XRAY SDK (true/false)                            |
-| EVENT_INTERVAL_SECONDS | Number of seconds to wait before sending an event                   |
-
-The [run.sh](run.sh) script provided sets these environment variables and starts the app. If you wish to use it,
-make sure to edit it to include the missing environment variables (secrets and such).
 
 A makefile with various commands is provided for convenience and to ease development. These are:
 
-#### Install dependencies
-`make init`
-#### Format using black
-`make format`
-#### Show format diff using black
-`make format-diff`
-#### Lint using flake8
-`make lint`
-#### Run the app
-`make run-app`
-#### Execute the [run.sh](run.sh) script
-`make run-script`
+```sh
+$ make init # Install dependencies
+$ make format # Format using black
+$ make lint # Lint using flake8
+$ make run # Run application (as described below)
+```
 
-## Metrics
-This app uses the [prometheus_client](https://github.com/prometheus/client_python) library to expose
-metrics to Prometheus regarding pipeline latency through a http server on port 8000.
+### Running
+
+By issuing the command `make run`, a local environment is configured comprising of the following services (see `local-compose.yaml`):
+
+* [Prometheus](https://hub.docker.com/r/prom/prometheus)
+* [Grafana](https://hub.docker.com/r/grafana/grafana) (including [tns-db](https://hub.docker.com/r/grafana/tns-db) and provisioned datasource and dashboard for the application)
+* A websocket server that simply echoes received events to connected clients (emulating [event-data-subscription](https://github.com/oslokommune/event-data-subscription))
+* A web server which accepts post requests for events (emulating [okdata-event-collector](https://github.com/oslokommune/okdata-event-collector))
+
+By specifying `LOCAL_SERVICES_ONLY=true` when running the application (e.g. `LOCAL_SERVICES_ONLY=true make run`), the application bypasses connections to `event-collector`/ `event-data-subscription` and is instead configured to use the "dummy" WebSocket/HTTP servers. 
```

Comment:
Dette er kanskje det man vanligvis vil? I så fall gjøre det som default for `make run`? Eventuelt lage dedikerte make targets for dem, slik at man slipper å fikle med env-variablene?